.github/workflows/CompatHelper.yml

```
@@ -6,6 +6,8 @@ on:
 jobs:
   CompatHelper:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

```suggestion
    permissions:
      contents: write
```
Thanks to the unclear GitHub docs, I'm usure whether 
```yaml
permissions
   contents: write
   pull-requests: write
```
without the compathelper_priv key would be enough